src/mpb.cpp

```diff
@@ -440,7 +440,7 @@ void *fields::get_eigenmode(double frequency, direction d, const volume where, c
     // then, synchronize the data
     eps_data.ncache = eps_data.icache; // actual amount of cached data
     double *summed_cache = (double *)malloc(sizeof(double) * 6 * eps_data.ncache);
-    am_now_working_on(MpiTime);
+    am_now_working_on(MPI_all);
```

Comment:
I would use `MpiAll` or something like that, since `MPI_*` functions are names use by `mpi.h` itself.

Reply:
Better yet `MpiAllTime`